src/core/_template.scss

```diff
@@ -11,6 +11,15 @@
     // Prevent automatic text sizing, as we already cater for small devices and
     // would like the browser to stay on 100% text zoom by default.
     text-size-adjust: 100%;
+
+    // Force the scrollbar to always display, to prevent horizontal page jumps
+    // as content height changes (e.g. autocomplete results open). Screen only.
+    @include govuk-media-query($media-type: screen) {
+      overflow-y: scroll;
+
+      // Force IE10-11 scroll bar in "tablet" mode
```

Comment:
Your comments in the thread are really useful to understand why this is in here

"That said, there is no visual jump here as tablet mode IE draws the scroll bars on top of the content before fading away automatically"

I think it'd be good to add in the code too.

Reply:
But the more I read that, the more I think we shouldn't force scroll bars to be visible on "touch first" devices anymore.

Maybe at the launch of Windows 8 and the Microsoft Surface we were wrong to force them? We don't force them on Mac or iOS, or any other browser with auto-hiding scroll bars.

What do you think?

Reply:
As far as I can tell setting the overflow to scroll forces scrollbars to always be visible? Even on OSX?

Reply:
Not that I've seen.

I can only get scrollbars to appear on macOS (Chrome and Safari) by plugging in a scroll-wheel mouse on a device without a trackpad or magic mouse or by changing the "Show scroll bars" setting in System Preferences.

![scroll bars](https://user-images.githubusercontent.com/415517/53730762-c0001180-3e70-11e9-9a2d-aca37ff5dcb1.png)

Reply:
OK I mistakenly though when you used that CSS they were always visible.

Reply:
I get the sense it would be good to test this in browserstack against quite a few different devices. Do we need some test pages?

Reply:
Ah sorry, no, it's only Internet Explorer that seems to show a faded out disabled scroll bar once `overflow-y: scroll` is set. On macOS the default is also to auto-height, even with this set.

I'm going to remove this bit:

```scss
// Force IE10-11 scroll bar in "tablet" mode
-ms-overflow-style: scrollbar;
```

I'll add some more notes and a video below.

Reply:
Marking resolved as this line has been removed.